tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "jsx": "react",
     "target": "ES2016",
     "module": "commonjs",
-    "lib": ["ES2019.Array", "ES2019.Object"],
+    "lib": ["ES2019.Array", "ES2022.Object"],
```

Comment:
Curious if we can just change this to `ESNext`? Does it still compile okay?

Reply:
Yeah, rather than maintaining individual specifications, we just use one catch-all.